docs/SnapshotTesting.md

```diff
@@ -39,7 +39,11 @@ exports[`renders correctly 1`] = `
 `;
 ```
 
-The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process. Jest uses [pretty-format](https://github.com/facebook/jest/tree/master/packages/pretty-format) to make snapshots human-readable during code review. On subsequent test runs Jest will simply compare the rendered output with the previous snapshot. If they match, the test will pass. If they don't match, either the test runner found a bug in your code that should be fixed, or the implementation has changed and the snapshot needs to be updated.
+The snapshot artifact should be committed alongside code changes, and reviewed as part of your code review process. Jest uses [pretty-format](https://github.com/facebook/jest/tree/master/packages/pretty-format) to make snapshots human-readable during code review. On subsequent test runs Jest will simply compare the rendered output with the previous snapshot. If they match, the test will pass. If they don't match, either the test runner found a bug in your code (in this case, it's `Link` component ) that should be fixed, or the implementation has changed and the snapshot needs to be updated.
+
+> Note: The test will pass even if you remove `class` from the `<Link>` used in any other file.
```

Comment:
`class` is not used as a prop in Link component. Can you use e.g. `page`?
I think it could be phrased in a different way, what do you think:

> Note: The snapshot is directly scoped to the data you render – in our example it's `<Link />`  component with `page` prop passed to it. Rendering the same component with different props in other snapshot test will not affect the first one, as the tests don't know about each other.